### DISCUSSION TITLE
Notify about No-Participation links

### DISCUSSION THREAD

**Discussion by Wilker-uwu** [Docile-Alligator/Infinity-For-Reddit#912] [Ideas]
to make sure to notify in a clear way when the user enters a post or thread through a no-participation link (links using the subdomain `np.` in the Reddit domain), maybe with a banner at the top of the screen in a contrasting color, telling the user not to comment or vote in that thread, like it is done on Slider for example

**Comment by ghost** [comment]
Maybe even just disable the ability to comment and vote in no-participation links, instead of just displaying a warning?

**Comment by Wilker-uwu** [comment]
might be a good idea, or maybe also include an option to still allow commenting with a small inconvenience like a dialogue to confirm action via np links, but people here probably knows better than me \^\^\'

**Comment by gbakeman** [comment]
I agree that it would be nice to see some special handling of np links. Definitely a notification at the top of some kind, as well as warnings when attempting to participate.
